app/src/forms/common/models/tables/form.js

```diff
@@ -118,6 +117,7 @@ class Form extends Timestamps(Model) {
       'name',
       'description',
       'active',
+      'allowSubmitterToUploadFile',
```

Comment:
This isn't just submitters that can upload files. You must check and make sure that only users that are logged in can upload files. Public anonymous forms have the file upload feature disabled.

Reply:
We did that.

Reply:
allowSubmitterToUploadFile is not for submitter is for the form permission to upload file , any one who got submitter permission can upload file.